config/web.js

```diff
@@ -195,6 +195,12 @@ module.exports = {
   },
 
   options: {
+    info: {
+      // We always want access to request.info.remoteAddress in hapi@19+
+      //
+      // @see https://github.com/hapijs/hapi/issues/4017
+      remote: true,
+    },
```

Comment:
This fixed the breaking change and retained the pre-v19 behavior of always populating `request.info.remoteAddress` even after the request is aborted.